cpc/cpc_default_config.py

```diff
@@ -64,7 +64,7 @@ def set_default_cpc_config(parser):
                        choices=['reverse', 'none'],
                        help='Some variations on CPC.')
     group.add_argument('--encoder_type', type=str,
-                       choices=['cpc', 'mfcc', 'lfb'],
+                       choices=['cpc', 'mfcc', 'lfb', 'smart'],
```

Comment:
Remove this, this is for smartpooling which we do not use in this work

Reply:
done